docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -270,8 +272,14 @@ appropriate flags on the Trac ticket based on the results of your review:
 "Patch needs improvement", "Needs documentation", and/or "Needs tests". As time
 and interest permits, mergers do final reviews of "Ready for checkin" tickets
 and will either commit the changes or bump it back to "Accepted" if further
-work needs to be done. If you're looking to become a merger, doing thorough
-reviews of contributions is a great way to earn trust.
+work needs to be done.
+
+.. note::
+    Ensure that a ticket has been submitted and accepted before proceeding
+    with the review.
```

Comment:
Thanks, but I can also imagine leaving this section as it was. Let's see what others think.

Reply:
I think I agree as the second sentence onwards has the assumption that there is a ticket.

If we make a change here, I would update:
```
Use this checklist to review a pull request. If you are reviewing a pull
request that is not your own and it passes all the criteria below, please set
the "Triage Stage" on the corresponding Trac ticket to "Ready for checkin".
```
to something like
```
Use this checklist to review a pull request. If you are reviewing a non-trivial
pull request, first ensure the it has an accepted ticket. If the pull passes all 
the criteria below and is not your own, please set the "Triage Stage" on the 
corresponding Trac ticket to "Ready for checkin".
```
----

I noticed that these docs have the following sections:
1. "Typo fixes and trivial documentation changes": to me, this covers the contributions where we **do not expect a ticket**, pretty much all others we would expect an accepted ticket
2. "Non-trivial contributions": to me, this would normally mean contributions where we **expect a discussion** which concludes in an accepted ticket. However, in this context this seems to be saying that although we have accepted the ticket we still expect a discussion (whether that's the same or a new one) around the approach. We are not strict on this as a requirement, as alternative approaches are usually discussed in the review itself. This also doesn't mention DEPs and I'm not sure if that is what it is going for?

Unfortunately, that we have these two sections on the same page means that it's not very clear what we are talking about when mentioning a "non-trivial" thing.

I don't have the answer here but I would like us to link to the "Typo fixes and trivial documentation changes" section and when talking about a "non-trivial pull request" 👍 

----

Quite unrelated
> If you're looking to become a merger, doing thorough reviews of contributions is a great way to earn trust.

Perhaps "merger" might want to be updated to something like "member of the Review and Triage team" or something very generic like "hold a position of responsibility in Django". 
I appreciate that this is a separate concern to what this PR is addressing though 👍